src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -2830,15 +2820,12 @@ export default class EthereumApi implements Api {
    */
   @assertArgLength(2, 3)
   async personal_unlockAccount(
-    address: DATA,
+    strAddress: DATA,
```

Comment:
The name will be displayed in the interactive docs, so I think it should remain as `address`.

Reply:
Good find!